scalability_and_performance/dummy-assembly.adoc

```diff
@@ -0,0 +1,23 @@
+[id="cnf-create-performance-profiles]
```

Comment:
```suggestion
:_content-type: ASSEMBLY
[id="cnf-create-performance-profiles]
```

Reply:
The content type for the file. Replace <TYPE> with the actual type of the module, CONCEPT, REFERENCE, or PROCEDURE. Place this attribute before the anchor ID or, if present, the conditional that contains the anchor ID. 

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata